malaria24/ona/tasks.py

```diff
@@ -140,6 +144,16 @@ def make_pdf(html_content):  # pragma: no cover
     return value
 
 
+@celery_app.task(ignore_result=True)
+def compile_and_send_jembi(case_pk):
+    case = ReportedCase.objects.get(pk=case_pk)
+    api_url = 'http://jembi.org/malaria24'
+    case_dictionary = case.get_data()
+    auth = HTTPBasicAuth(USER_NAME, PASSWORD)
+    r = requests.post(api_url, json=case_dictionary, auth=auth)
+    print r.json
```

Comment:
I'm not sure if print is the best idea here, have a look at this example, it will handle the errors that might happen and even retry if possible:
https://github.com/praekeltfoundation/ndoh-hub/blob/develop/registrations/tasks.py#L582

Reply:
Took a look at it, Thank you